Source/Mods/RPGStyleInventory.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using HarmonyLib;
+using Multiplayer.API;
+using Verse;
+using RimWorld;
+
+namespace Multiplayer.Compat
+{
+    /// <summary>RPG Style Inventory by Sandy</summary>
+    /// <see href="https://steamcommunity.com/sharedfiles/filedetails/?id=1561221991"/>
+    [MpCompatFor("Sandy.RPGStyleInventory")]
+    class RPGStyleInventory
+    {
+
+
+        public RPGStyleInventory(ModContentPack mod)
+        {
+            MP.RegisterSyncWorker<ITab_Pawn_Gear> (SyncITab, AccessTools.TypeByName("Sandy_Detailed_RPG_Inventory.Sandy_Detailed_RPG_GearTab"));
+            MP.RegisterSyncMethod(AccessTools.TypeByName("Sandy_Detailed_RPG_Inventory.Sandy_Detailed_RPG_GearTab"), "InterfaceDrop").SetContext(SyncContext.MapSelected);
+            MP.RegisterSyncMethod(AccessTools.TypeByName("Sandy_Detailed_RPG_Inventory.Sandy_Detailed_RPG_GearTab"), "InterfaceIngest").SetContext(SyncContext.MapSelected);
+
```

Comment:
Hold on, I asked for it to be made into a local variable so it doesn't linger around.

Reply:
Ah ok i will fix it right away

Reply:
OK fixed